src/zui/ZUIHeader.tsx

```diff
@@ -53,11 +53,12 @@ const useStyles = makeStyles<Theme, StyleProps>((theme) => ({
 }));
 
 interface HeaderProps {
-  actionButtons?: React.ReactElement | React.ReactElement[];
+  actionButtons?: ReactElement | ReactElement[];
   avatar?: string;
   collapsed?: boolean;
   ellipsisMenuItems?: ZUIEllipsisMenuProps['items'];
   onToggleCollapsed?: (collapsed: boolean) => void;
+  datePicker?: ReactElement;
```

Comment:
IMO this is too specific of a name for this area of the header. The other areas are "title", "subtitle" and "action buttons", all of which have names that can be used quite generally.

The area designated by this new property is only used for one purpose right now, but if we really only wanted to use it for one thing, we might as well build the datepicker into the header.

If we don't want to lock it down so strictly to being just for date pickers, I think the name should be something else.

What do you think?

Reply:
"belowActionButtons", "rightSideInfo", "actionButtonSubtitle", "rightSideSubtitle" ?